modules/tensor_mechanics/src/actions/DomainIntegralAction.C

```diff
@@ -68,6 +71,13 @@ validParams<DomainIntegralAction>()
       false,
       "Calculate an equivalent K from KI, KII and KIII, assuming self-similar crack growth.");
   params.addParam<bool>("output_q", true, "Output q");
+  params.addParam<bool>("solid_mechanics",
+                        false,
+                        "Set to true if the solid_mechanics system is "
+                        "used. This option is only needed for "
+                        "interaction integrals.");
+  params.addParam<std::vector<MaterialPropertyName>>(
+      "eigenstrain_names", "List of eigenstrains to be applied in this strain calculation");
```

Comment:
List of eigenstrains applied in the strain calculation

Reply:
I have corrected this parameter description.